indexing-service/src/main/java/org/apache/druid/indexing/common/actions/RetrieveUnusedSegmentsAction.java

```diff
@@ -40,6 +40,9 @@ public class RetrieveUnusedSegmentsAction implements TaskAction<List<DataSegment
   @JsonIgnore
   private final Interval interval;
 
+  @JsonIgnore
```

Comment:
Do all of these fields need the `@JsonIgnore` tag? Seems a little misleading since the fields are actually meant to be serialized.

Reply:
Good question. On second thought, it's confusing. I opened a PR to remove these annotations from the different task action classes - https://github.com/apache/druid/pull/16099

Reply:
So, you plan to remove the `@JsonIgnore` from here too, right?

Reply:
Yeah, it's going to conflict with https://github.com/apache/druid/pull/16099, so I will merge that change in this branch and then remove it here